projects/rizin/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM sweetvishnya/ubuntu20.04-sydr-fuzz
+
+# Set author of this docker container.
+LABEL maintainer="Theodor Arsenij Larionov"
+
+# Install dependencies for build and analysis.
+RUN apt-get update && \
+    apt-get -y install autoconf automake libtool ninja-build
+
+ENV PATH="${PATH}:/root/.local/bin"
+
+# Install meson
+RUN pip3 install --user meson==0.63.0
+
+# Clone target and corpus from github.
+RUN mkdir rizin-fuzzing && cd rizin-fuzzing && git clone https://github.com/rizinorg/rizin
```

Comment:
We may clone rizin in root /rizin

Reply:
I think it's better to have separate directory for all fuzzing-related things (binaries, source code, etc)